### PR TITLE
CAT-1240 License Text to 2015

### PR DESCRIPTION
**Pull request by bernadettespieler**
ok to test

### CODE REVIEW

**Comment by obusher** [comment]
Please do not commit local config files such as `.idea/*` and `*.iml`. :ice_cream:

File: build.gradle [hunk 1]
```diff
@@ -39,7 +39,7 @@ ext {
             "formula_editor_lists"              : false,
             "physics_engine_collision_filtering": false,
             "time_capsule"                      : false,
-            "userbricks"                        : false
+            "userbricks"                        : true
```
Comment (obusher) [inline build.gradle]: You really wanna open pandoras box? :O

**Comment by thmq** [comment]
i will revert the changes to the files that must not be named ... i mean changed ...

**Comment by thmq** [comment]
ok to test

**Comment by linsm** [comment]
please reopen your pull request

use your forked repository
target of pull request should be 'develop' branch
link THIS pull request in the new one

**Pull request closed without merging**